subprojects/packagefiles/uchardet/src/meson.build

```diff
@@ -0,0 +1,72 @@
+uchardet_sources = files(
+  'CharDistribution.cpp',
+  'JpCntx.cpp',
+  'LangModels/LangArabicModel.cpp',
+  'LangModels/LangBulgarianModel.cpp',
+  'LangModels/LangCroatianModel.cpp',
+  'LangModels/LangCzechModel.cpp',
+  'LangModels/LangEsperantoModel.cpp',
+  'LangModels/LangEstonianModel.cpp',
+  'LangModels/LangFinnishModel.cpp',
+  'LangModels/LangFrenchModel.cpp',
+  'LangModels/LangDanishModel.cpp',
+  'LangModels/LangGermanModel.cpp',
+  'LangModels/LangGreekModel.cpp',
+  'LangModels/LangHungarianModel.cpp',
+  'LangModels/LangHebrewModel.cpp',
+  'LangModels/LangIrishModel.cpp',
+  'LangModels/LangItalianModel.cpp',
+  'LangModels/LangLithuanianModel.cpp',
+  'LangModels/LangLatvianModel.cpp',
+  'LangModels/LangMalteseModel.cpp',
+  'LangModels/LangPolishModel.cpp',
+  'LangModels/LangPortugueseModel.cpp',
+  'LangModels/LangRomanianModel.cpp',
+  'LangModels/LangRussianModel.cpp',
+  'LangModels/LangSlovakModel.cpp',
+  'LangModels/LangSloveneModel.cpp',
+  'LangModels/LangSwedishModel.cpp',
+  'LangModels/LangSpanishModel.cpp',
+  'LangModels/LangThaiModel.cpp',
+  'LangModels/LangTurkishModel.cpp',
+  'LangModels/LangVietnameseModel.cpp',
+  'nsHebrewProber.cpp',
+  'nsCharSetProber.cpp',
+  'nsBig5Prober.cpp',
+  'nsEUCJPProber.cpp',
+  'nsEUCKRProber.cpp',
+  'nsEUCTWProber.cpp',
+  'nsEscCharsetProber.cpp',
+  'nsEscSM.cpp',
+  'nsGB2312Prober.cpp',
+  'nsMBCSGroupProber.cpp',
+  'nsMBCSSM.cpp',
+  'nsSBCSGroupProber.cpp',
+  'nsSBCharSetProber.cpp',
+  'nsSJISProber.cpp',
+  'nsUTF8Prober.cpp',
+  'nsLatin1Prober.cpp',
+  'nsUniversalDetector.cpp',
+  'uchardet.cpp',
+)
+
+uchardet_inc = [
```

Comment:
erm. include_directories is a variadic function. is there a need to wrap it in an array?